spec/spec_helper.rb

```diff
@@ -2,7 +2,21 @@
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 
 require 'coveralls'
-Coveralls.wear!
+require 'simplecov'
+require 'simplecov-console'
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+    Coveralls::SimpleCov::Formatter,
+    SimpleCov::Formatter::HTMLFormatter,
+    SimpleCov::Formatter::Console,
+]
+SimpleCov.start do
+  add_filter %r{/spec/}
+  add_filter 'lib/watir/elements/html_elements.rb'
+  add_filter 'lib/watir/elements/svg_elements.rb'
+  add_filter 'lib/watir/legacy_wait.rb'
+  add_filter 'lib/watirspec'
+  refuse_coverage_drop
```

Comment:
Would this mean that my pull request in #791 would fail? It has an overall coverage drop of -0.05%. Seems a bit extreme given I don't think my change actually dropped the coverage.

Maybe we should use maximum_coverage_drop instead? (https://coderwall.com/p/ki2sdq/simplecov-prevent-coverage-drop).

Reply:
No, because the two things that it reported as "dropping coverage" (which admittedly I don't understand why) are filtered out with the code above it.

The goal with it is to encourage people submitting PR to add a test even if they are only adding a line of code that isn't already being exercised.

Let's try it and change it if it does actually become too much. I think this update with the filters will remove false alarms, but we can see.